packages/node/test/machine/integration/protocols/install-uninstall.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { OutcomeType } from "@counterfactual/types";
+import { Two, Zero } from "ethers/constants";
+
+import { CONVENTION_FOR_ETH_TOKEN_ADDRESS } from "../../../../src/constants";
+import { toBeEq } from "../bignumber-jest-matcher";
+
+import { Participant, TestRunner } from "./test-runner";
+
+expect.extend({ toBeEq });
+
+export enum TokenType {
```

Comment:
This enum seems to be for the test suite only, maybe give it a more explicit name